napari/_tests/test_napari.py

```diff
@@ -10,31 +11,20 @@
 
 
 @pytest.mark.parametrize('layer_type, data, ndim', layer_test_data)
-def test_view(qtbot, layer_type, data, ndim):
-
+def test_view(qapp, layer_type, data, ndim):
     np.random.seed(0)
     viewer = view_layer_type(layer_type, data)
```

Comment:
suggestion: make `view_layer_type` a factory that cleans up the viewers that get created.

Reply:
since it's only used twice, and only in one module, I think i'd like to delay creating a new fixture for that.  they are cleaned with the viewer.close() at the end of the test